11/mysql-alpine-tomcat/Dockerfile

```diff
@@ -0,0 +1,103 @@
+# ---------------------------------------------------------------------------
+# See the NOTICE file distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as
+# published by the Free Software Foundation; either version 2.1 of
+# the License, or (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this software; if not, write to the Free
+# Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+# 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+# ---------------------------------------------------------------------------
+FROM tomcat:8-jre8-alpine
+
+#    ____  ____  ____      ____  _   __        _
+#   |_  _||_  _||_  _|    |_  _|(_) [  |  _   (_)
+#     \ \  / /    \ \  /\  / /  __   | | / ]  __
+#      > `' <      \ \/  \/ /  [  |  | '' <  [  |
+#    _/ /'`\ \_     \  /\  /    | |  | |`\ \  | |
+#   |____||____|     \/  \/    [___][__|  \_][___]
+
+MAINTAINER Vincent Massol <vincent@massol.net>
```

Comment:
This should be changed as I'm not planning to be the maintainer for it ;) @ASHISH932 Since you're contributing this are you ok to also maintain it on the long run? Thx